resources/assets/js/features/user/components/Login/Form.js

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../actions';
+import TextInput from './TextInput';
+import { bindActionCreators } from 'redux';
+import '../../styles/user.scss';
+
+class Form extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = { credentials: { email: '', password: '' }};
+        this.onChange = this.onChange.bind(this);
+        this.onSave = this.onSave.bind(this);
+        this.pickErrorMessage = this.pickErrorMessage.bind(this);
+    }
+
+    pickErrorMessage(code) {
+        switch(code) {
+            case 401:
+                return (<div>Your e-mail was not activated yet. <a href="/verification">Click to proceed with verification</a></div>)
+            case 404:
+                return (<div>User with such e-mail wasn't registered yet. <a href="/registration"> Goto to registration page </a></div>)
+            case 422:
+                return (<div>E-mail and password are not corressponding each other</div>)
+            default:
+                return (<div>Some problems appeared. Try once more in a moment</div>)
+        }
+    }
+
+    onChange(event) {
+        const field = event.target.name;
+        const credentials = this.state.credentials;
+        credentials[field] = event.target.value;
+
+        return this.setState({ credentials: credentials });
+    }
+
+    onSave(event) {
+        event.preventDefault();
+        this.props.actions.doLogin(this.state.credentials);
```

Comment:
on this stage field data should be validated, and if validation successful - send request or call action